src/views/main/request.hbs

```diff
@@ -0,0 +1,85 @@
+<main>
+  <section class="nav--form">
+    <div class="nav--form--sections">Personal Details</div>
+    <div class="nav--form--sections">Your Request</div>
+  </section>
+  <section class="personal--details">
+    <form>
+      <label for="full--name" class="personal--details--label">Full Name :</label>
+      <input
+        type="text"
+        placeholder="Enter Your Full Name ... "
+        id="full--name"
+        class="full--name--form input"
+      />
+
+      <label for="phone--number" class="personal--details--label">Phone Number :</label>
+      <input
+        type="text"
+        placeholder="Enter Your Phone Nmuber ... "
+        id="phone--number"
+        class="phone--number--form input"
+      />
+
+      <label for="city--name" class="personal--details--label">City Name :</label>
+      <input
+        type="text"
+        placeholder="Enter Your City Name ... "
+        id="city--name"
+        class="city--nam--form input"
+      />
+
+      <label for="street--name" class="personal--details--label">Street Name :</label>
+      <input
+        type="text"
+        placeholder="Enter Your City Name ... "
+        id="street--name"
+        class="street--name--form input"
+      />
+      <input
+        type="button"
+        name="Next"
+        value="Next"
+        id="next--button"
+        class="next--button"
+      />
+    </form>
+  </section>
+  <section class="your--request">
+    <form>
+      <label for="number--of--pieces" class="your--request--label">Number of pieces :</label>
+      <input
+        type="text"
+        placeholder="Enter number of pieces ... "
+        id="number--of--pieces"
+        class="number--of--pieces input"
+      />
+      <div>
+        <label for="gender">Gender :</label>
+        <input type="checkbox" name="genderWomen" class="gender" id="women" /><lable for="women">Women</lable>
+        <input type="checkbox" name="genderMen" class="gender" id="men"/> <lable for="men">Men</lable>
+        <input type="checkbox" name="genderKids" class="gender" id="kids"/><label for="kids">Kids</label>
+      </div>
+      <div>
+          <label for="season">Season :</label>
+          <input type="checkbox" name="seasonWinter" class="season" /><label for="winter">Winter</label>
```

Comment:
Associate each label  with a given input by using the attribute for or
by putting the input inside the label but  it much less flexible when styling with CSS